app/Console/Commands/ExternalServices/ManageDiscord.php

```diff
@@ -100,13 +108,17 @@ protected function grantRoles()
         });
     }
 
-    protected function removeRoles()
+    public function removeRoles()
     {
         $account = $this->account;
         $discord = $this->discord;
 
         $currentRoles = $discord->getUserRoles($account);
 
+        if (! $account->isBanned && $currentRoles->contains($this->suspendedRoleId)) {
+            $discord->removeRoleById($account, $this->suspendedRoleId);
```

Comment:
Could we somehow, within this block, also remove any other roles that they have in `$currentRoles`? Regardless of what `DiscordRole` says they should have?

We wouldn't need to worry about giving them the roles back after being unbanned as that would be picked up by lines 101 onwards?

Reply:
Actually, this would be in the `grantRoles` block, perhaps by calling a `$this->removeAllRoles()` method?